core/templates/cards.html

```diff
@@ -131,36 +130,31 @@ <h4 class="modal-title">Face Off</h4>
         else{
             if(gameMode != easyGame)
                 startHardGame();
-
             else
                 startEasyGame();
-
         }
     });
-
-
     function showGameMenu(){
         $('#gameMenu').removeClass('invisible');
         $('#noPicMenu').addClass('invisible');
         $('#noPicText').addClass('invisible');
         $('#addGameText').removeClass('invisible');
     }
-
     function setForm(){
         $('[name=score]').val(score);
         $('[name=cardIndex]').val(selectedCardIndex);
         $('[name=results]').val('{{ cards }}');
         $('#resultsForm').submit();
     }
-
     function startEasyGame(){
         gameMode = easyGame;
+        startTimer();
+        $("#countdown").addClass("invisible");
     }
```

Comment:
so we still want to remove the timer except now we want to refactor some of the Javascript logic to account for this. You'll see in the startTimer() method it calls the 'disableCards' function when it completes and this renders the next_round button. 

Now we want the next round button to render when the card flips. this would be the best place to insert this logic based on the "gameMode"